go/vt/mysqlctl/xtrabackupengine.go

```diff
@@ -80,7 +79,11 @@ const (
 type xtraBackupManifest struct {
 	// BackupManifest is an anonymous embedding of the base manifest struct.
 	BackupManifest
-
+	// CompressionEngine stores which compression engine was originally provided
+	// to compress the files. Please note that if user has provided externalCompressorCmd
+	// then it will contain value 'external'. This field is used during restore routine to
```

Comment:
Would it also be helpful to store the external compressor command in the manifest? Even if it's not something Vitess can use, it might be helpful as an historical record for the user in case they end up in a scenario where they lose track of the command they used to compress.

Reply:
Yes I thought of doing it. I am assuming external compressor is not something people will use a lot. So i decided to keep it as a separate work item if needed in future. Let me know if you think otherwise. 
https://github.com/planetscale/vitess-private/issues/934

Reply:
Sounds like a plan!